api-catalog-ui/frontend/src/components/Wizard/configs/wizard_micronaut_categories.jsx

```diff
@@ -47,49 +50,58 @@ export const micronautSpecificCategories = [
                 value: '',
                 question: 'The alias used to address the private key in the keystore:',
                 dependencies: { enabled: true },
+                tooltip: '${fill.your.keyAlias}',
             },
             keyPassword: {
                 value: '',
                 question: 'The password associated with the private key:',
                 dependencies: { enabled: true },
                 type: 'password',
+                tooltip: '${fill.your.keyPassword}',
             },
             keyStore: {
                 value: '',
                 question: 'The keystore file used to store the private key (keyring: set to SAF keyring location):',
                 dependencies: { enabled: true },
+                tooltip: '${fill.your.keyStore}',
             },
             keyStorePassword: {
                 value: '',
                 question: 'The password used to unlock the keystore:',
                 dependencies: { enabled: true },
                 type: 'password',
+                tooltip: '${fill.your.keyStorePassword}',
             },
             keyStoreType: {
                 value: '',
                 question: 'Type of the keystore:',
                 dependencies: { enabled: true },
+                tooltip: '${fill.your.keystoretype}, e.g. PKCS12',
             },
             trustStore: {
                 value: '',
                 question: 'The truststore file used to keep other parties public keys and certificates:',
                 dependencies: { enabled: true },
+                tooltip: '${fill.your.trustStore}',
             },
             trustStorePassword: {
                 value: '',
                 question: 'The password used to unlock the truststore:',
                 dependencies: { enabled: true },
                 type: 'password',
+                tooltip: '${fill.your.trustStorePassword}',
             },
             trustStoreType: {
                 value: 'PKCS12',
                 question: 'Truststore type:',
                 dependencies: { enabled: true },
+                tooltip: 'Fill your trustStoreType',
             },
             ciphers: {
                 value: '',
                 question: 'SSL cipher suites:',
                 dependencies: { enabled: true },
+                tooltip: '${fill.your.ciphers}',
```

Comment:
Could you provide more information about ciphers? Could be: `Ciphers that are used by the HTTPS servers in API ML services. Ciphers can be externalized by specifying -Dapiml.security.ciphers command line parameter`, for instance.